File: 1/solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        occur = defaultdict()
        for i in range(len(nums)):
            if nums[i] not in occur:
                occur[target-nums[i]] = i
            else:
                return [occur[nums[i]], i]

File: 1/solution.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int, int> map;
        vector<int> ans;
        int len = nums.size();
        for(int i=0; i<len; ++i){
            if(map.find(target-nums[i]) == map.end()){
                map[nums[i]] = i;
                continue;
            }
            ans.push_back(map[target-nums[i]]);
            ans.push_back(i);
            return ans;       
        }
        return ans;
    }
};
